### DISCUSSION TITLE
在salvo中引入包就报错

### DISCUSSION THREAD

**Discussion by lopo1** [rbatis/rbatis#476] [General]
版本
rbatis = { version = "4.5"} 
编译的时候报错
functions in traits cannot be declared `async`
  --> C:\Users\81236\.cargo\registry\src\mirrors.tuna.tsinghua.edu.cn-df7c3c540f42cdbd\fast_pool-0.2.0\src\lib.rs:55:5
   |
55 |     async fn check(&self, conn: &mut Self::Connection) -> Result<(), Self::Error>;
   |     -----^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

**Comment by zhuxiujia** [comment]
> 版本 rbatis = { version = "4.5"} 编译的时候报错 functions in traits cannot be declared `async` --> C:\Users\81236.cargo\registry\src\mirrors.tuna.tsinghua.edu.cn-df7c3c540f42cdbd\fast_pool-0.2.0\src\lib.rs:55:5 | 55 | async fn check(&self, conn: &mut Self::Connection) -> Result<(), Self::Error>; | -----^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

你需要`rustup update`更新rust编译器版本，因为新版本RBatis使用了最新的async trait 特性，这需要rust 稳定版1.75版本以上。
版本发布说明提到 了这一点。
https://github.com/rbatis/rbatis/releases/tag/v4.5.10

**Comment by lopo1** [comment]
非常感谢
